### DISCUSSION TITLE
Holes in Sensor Lid Prints Question

### DISCUSSION THREAD

**Discussion by polygonfuture** [srozum/film_camera_tester#14] [Q&A]
Hey Serhiy,

Was wondering was there a specific reasoning behind not adding the 0.8mm holes to the printed lid model files?  I only ask because I converted the Gerber files to DXFs and made variation of Sensors 1 and 2 .STL files with 0.8mm holes already added.   

Most printers will print the holes slightly undersized, but then all a person needs to do is use a pin vise or drill bit to make the final hole.  It would cut out any room for error of placement.  With a pin vise and 0.8mm bit it takes all of 1 second to make a perfect 0.8mm hole.  I was thinking of doing a pull request so these could be added to the repo but wanted to get your thoughts on this.

If this is something you'd be interested in allowing then I would go ahead and do it for Sensor 6 as well and then submit a PR.

Let me know your thoughts.

**Comment by polygonfuture** [comment]
Oh also to add I could include a Rhino or Fusion file that includes the Sensor .stl files and DXF file outlines so users can verify placement.

**Comment by srozum** [comment]
I do have STLs with holes, I did tried to print them and wasn't satisfied with results. So instead went with "PCB as a stencil", and found it more precise, and at the same time more useful during assembly process. Because same PCB is used to drill holes, bent photodiode pins, align photodiodes.

Maybe it would work better for resin printers. So, sure, create a PR. Thanks
